update.sh

```diff
@@ -22,6 +22,7 @@ rubygems='3.0.3'
 declare -A newEnoughRubygems=(
 	[2.6]=1 # 2.6.3 => gems 3.0.3 (https://github.com/ruby/ruby/blob/v2_6_3/lib/rubygems.rb#L12)
 	[2.7]=1 # 2.7.0-preview2 => gems 3.1.0.pre1 (https://github.com/ruby/ruby/blob/v2_7_0_preview1/lib/rubygems.rb#L12)
+	[3.0]=1 # 3.0.0-preview1 => gems 3.2.0.rc.1 (https://github.com/ruby/ruby/blob/v3_0_0_preview1/lib/rubygems.rb#L11)
```

Comment:
This is *verbatim* what I have in my local working directory where I was experimenting with making this change. :sweat_smile: :heart_eyes: